pkg/apiserver/default_storage_factory_builder.go

```diff
@@ -55,15 +54,6 @@ func mergeResourceEncodingConfigs(defaultResourceEncoding *serverstorage.Default
 	return resourceEncodingConfig
 }
 
-// Merges the given defaultResourceConfig with specific GroupVersion overrides.
-func mergeGroupEncodingConfigs(defaultResourceEncoding *serverstorage.DefaultResourceEncodingConfig, storageEncodingOverrides map[string]schema.GroupVersion) *serverstorage.DefaultResourceEncodingConfig {
-	resourceEncodingConfig := defaultResourceEncoding
-	for group, storageEncodingVersion := range storageEncodingOverrides {
-		resourceEncodingConfig.SetVersionEncoding(group, storageEncodingVersion, schema.GroupVersion{Group: group, Version: runtime.APIVersionInternal})
```

Comment:
SetVersionEncoding was straight up removed. I don't see an equivalent.

Reply:
that should be fine.